sklvq/models/_gmlvq.py

```diff
@@ -385,7 +386,7 @@ def _normalize_omega(omega: np.ndarray) -> None:
 
     def _correct_omega(self, omega: np.ndarray) -> None:
         # Note matmul is faster for larger matrices, for smaller dot is faster.
-        np.matmul(omega.T, self.relevance_correction, out=omega.T)
+        np.matmul(self.relevance_correction,omega,out=omega.T)       
```

Comment:
Are you sure this is correct and you did not change the way the correction matrix was defined?